src/chrome/content/rules/Jimdo.design.xml

```diff
@@ -0,0 +1,20 @@
+
+<!--
+	Other Jimdo rulesets:
+		-Jimdo.xml
+		-Jimdo-Server.com.xml
+		-Jimcdn.com.xml
+		-Jimstatic.com.xml
+-->
+<ruleset name="jimdo.design (partial)">
```

Comment:
Use `Jimdo.design` to match the filename. We also don't need the `(partial)` tag.